qa/sc_evm_sync_status_forger_node_corner_case.py

```diff
@@ -0,0 +1,234 @@
+#!/usr/bin/env python3
```

Comment:
Is there a specific reason why we didn't include this test in the **run_sc_test.sh** script?

Reply:
No specific reason, it was an oversight. Test added to the `run_sc_test.sh` script.
I also change the python test name to a more generic `sc_evm_sync_status_forger_nodes.py` in case we need to add more test cases in the future that involve more forger nodes.